app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -40,6 +40,60 @@ function DocumentCapture({ onStepChange = () => {} }: DocumentCaptureProps) {
   const { inPersonFullAddressEntryEnabled, inPersonURL, skipDocAuth, skipDocAuthFromHandoff } =
     useContext(InPersonContext);
   const appName = getConfigValue('appName');
+  const inPersonLocationPostOfficeSearchForm = inPersonFullAddressEntryEnabled
```

Comment:
I think there's another ticket floating around out there that has us removing the address code altogether, since it's unused. That'd make this refactoring even better. Dunno that it's necessarily a blocker, since it'd hopefully simplify the code in the future.

Reply:
Going to keep it in and bring up your comment in my stand up

Reply:
It looks like [LG-10605](https://cm-jira.usa.gov/browse/LG-10605) was the ticket mentioned when this was [brought up in Slack recently](https://gsa-tts.slack.com/archives/C04GA9WQ85B/p1719844684712719).

Reply:
@aduth would that essentially be removing the locationFormStep variable and removing its reference in the inPersonSteps array (Line 132)?

Reply:
I think the future state I imagined was that we'd have either a post office search or full address entry, but not continue to maintain both implementations. So the ternary on this line would become unnecessary.

It seems like the future of the search feature isn't very settled at the moment, so I wouldn't worry too much about it for now.